### ISSUE TITLE
.net core 3 preview 7. Could not load type 'System.Text.Json.Serialization.JsonSerializer' from assembly 'System.Text.Json'

### ISSUE THREAD

**Issue by aford22** [Blazored/LocalStorage#19]
Using .net core 3 preview 7. Getting exception:

```
System.TypeLoadException: Could not load type 'System.Text.Json.Serialization.JsonSerializer' from assembly 'System.Text.Json, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
   at async Task<T> Blazored.LocalStorage.LocalStorageService.GetItemAsync<T>(string key)
   at void System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start<TStateMachine>(ref TStateMachine stateMachine)
   at void System.Runtime.CompilerServices.AsyncTaskMethodBuilder<TResult>.Start<TStateMachine>(ref TStateMachine stateMachine)
   at Task<T> Blazored.LocalStorage.LocalStorageService.GetItemAsync<T>(string key)
```

**Comment by chrissainty** [comment]
Yes, this is because the namespace and method names have changed for the new JSON serializer in Preview 7. This will be addressed when Local Storage has been upgraded to Preview 7.

**Comment by chrissainty** [comment]
This is now fixed by #21

**Comment by aford22** [comment]
Good man. Thanks Chris
